File: 54.spiral-matrix/TestData.java
import java.util.*;

class TestData {

    public static void main(String[] args) {
        SpiralMatrixLeetcode solution = new SpiralMatrixLeetcode();;
        Data data = new Data();;

        for (int i = 0; i < data.inputs.length; i++) 
            if (!solution.spiralOrder(data.inputs[i]).equals(Arrays.asList(data.outputs[i]))) {
                System.out.println("Fail in test: " + i);
                System.exit(0);
            }    
                   
        System.out.println("OK!");
    }
}

File: 54.spiral-matrix/test_data.py
from spiral_matrix_leetcode import *
# from spiral_matrix_my import *
from data import *

if __name__ == "__main__":
    solution = Solution()

    for i in range(len(inputs)):
        if solution.spiralOrder(inputs[i]) != outputs[i]:
            print("Fail in test: ", i)
            exit(0)

    print("OK!")
